Neos.Flow/Classes/Core/Booting/Scripts.php

```diff
@@ -577,8 +578,10 @@ public static function initializeSystemFileMonitor(Bootstrap $bootstrap)
             if (!in_array($packageKey, $packagesWithConfiguredObjects)) {
                 continue;
             }
-            foreach ($package->getAutoloadPaths() as $autoloadPath) {
-                self::monitorDirectoryIfItExists($fileMonitors['Flow_ClassFiles'], $autoloadPath, '\.php$');
+            if ($package instanceof GenericPackage) {
```

Comment:
nothing you need to fix here, but we should probably add getAutoloadPaths() to the PackageInterface...